core/src/main/kotlin/ml/PositionEvaluation.kt

```diff
@@ -0,0 +1,128 @@
+package ml
+
+import core.CheckersModel
+import core.Color
+import core.FigureType
+
+class PositionEvaluation(val model: CheckersModel, val color: Color) {
+    val params = calc()
+
+    val checkers = Pair(params[0], params[1])
+    val queens = Pair(params[2], params[3])
+    val mainDiagQueen = Pair(params[10], params[11])
+    val averageVertical = Pair(if (checkers.first != 0) 1 + params[6] / (1.0 * checkers.first) else 0,
+                                if (checkers.second != 0) 1 + params[7] / (1.0 * checkers.second) else 0)
+    val averageHorizontal = Pair(if (checkers.first != 0) 1 + params[8] / (1.0 * checkers.first) else 0,
+                                if (checkers.second != 0) 1 + params[9] / (1.0 * checkers.second) else 0)
+    val safeCheckers = Pair(params[4], params[5])
+    val twoDiag = Pair(params[12], params[13])
+    val threeDiag = Pair(params[14], params[15])
+    val triangles = Pair(params[16], params[17])
+    val flippedTriangles = Pair(params[18], params[19])
+
+    lateinit var c5_f4: Pair<Int, Int>
+    lateinit var e5_d4: Pair<Int, Int>
+    lateinit var a7_h2: Pair<Int, Int>
+    lateinit var a3_h6: Pair<Int, Int>
+    lateinit var h4_a5: Pair<Int, Int>
+
+    private fun calc() : IntArray {
+        val ans = IntArray(30) {0}
+        /**
+         * ans[2 * i] for color player, ans[2 * i + 1] for adversary
```

Comment:
Ok, but it could be a structure instead, I think. Or array of scructures

Reply:
Yes, we have it in class, not in function